b2sdk/api.py

```diff
@@ -299,6 +299,10 @@ def delete_bucket(self, bucket):
         :type bucket: b2sdk.bucket.Bucket
         """
         account_id = self.account_info.get_account_id()
+        response = self.session.list_buckets(account_id)
```

Comment:
Before deleting bucket, fetching bucket information and rebuild the cache so that from other device, it can have transparency while deleting buckets created from other device